apps/meteor/tests/e2e/omnichannel/omnichannel-agents.spec.ts

```diff
@@ -105,4 +105,27 @@ test.describe.serial('OC - Manage Agents', () => {
 			await poOmnichannelAgents.btnSave.click();
 		});
 	});
+
+	test('OC - Edit agent  - Manage departments', async ({ page }) => {
+		await poOmnichannelAgents.selectUsername('user1');
+		await poOmnichannelAgents.btnAdd.click();
+		await poOmnichannelAgents.inputSearch.fill('user1');
+		await poOmnichannelAgents.findRowByUsername('user1').click();
+
+		await poOmnichannelAgents.btnEdit.click();
+		await poOmnichannelAgents.selectDepartment(department.data.name);
+		await poOmnichannelAgents.btnSave.click();
+
+		await test.step('expect the selected department is visible', async () => {
+			await poOmnichannelAgents.findRowByUsername('user1').click();
+
+			// mock the endpoint to use the one without pagination
+			await page.route(new RegExp('/api/v1/livechat/department.*'), async (route) => {
```

Comment:
Wouldn't this mock all the endpoints `department.`? I think this could be a problem when running tests concurrently.